client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,322 @@
+	overflow: hidden;
+	padding-left: 8px;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+
+	span {
+		color: $gray-text;
+		margin-right: 8px;
+	}
+
+	a {
+		color: $gray;
+		cursor: pointer;
+		fill: $gray;
+		&:focus,
+		&:hover {
+			color: $blue-wordpress;
+			fill: $blue-wordpress;
+		}
+	}
+}
+
+.comment-detail__comment {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	padding-left: 16px;
+}
+
+.comment-detail__comment-content {
+	border-left: 3px solid lighten( $gray, 30% );
+}
+
+.comment-detail__comment-body {
+	padding: 0 16px;
+}
+
+.comment-detail__comment-reply {
+	padding: 16px;
+
+	a {
+		color: $gray;
+		cursor: pointer;
+		fill: $gray;
+		&:focus,
+		&:hover {
+			color: $blue-wordpress;
+			fill: $blue-wordpress;
+		}
+	}
+}
+
+.comment-detail__author {
+	padding: 16px 0;
+
+	.comment-detail__author-info {
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+	.comment-detail__author-info-element {
+		overflow: hidden;
+		text-overflow: ellipsis;
+		white-space: nowrap;
+	}
+}
+
+.comment-detail__author-preview {
+	align-items: center;
+	display: flex;
+	flex-flow: row nowrap;
+	padding: 0 16px;
+	transition: padding-bottom .3s linear;
+
+	.comment-detail__author-avatar {
+		height: 40px;
+		width: 40px;
+	}
+
+	.comment-detail__author-info {
+		line-height: 1.4;
+		padding: 0 8px;
+	}
+}
+.comment-detail.is-expanded .comment-detail__author-preview {
+	padding-bottom: 16px;
+}
+
+.comment-detail__author-more-info-toggle {
+	color: $gray;
+	cursor: pointer;
+	display: block;
+	fill: $gray;
+	height: 24px;
+	margin-left: auto;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail__author-more-info {
+	border-bottom: 1px solid lighten( $gray, 30% );
+	color: $gray;
+	display: none;
+	transform: scaleY( 0 );
+	transform-origin: top;
+}
+.comment-detail__author.is-expanded .comment-detail__author-more-info {
+	animation: comment-detail__author-more-info .15s linear;
+	display: block;
+	transform: scaleY( 1 );
+}
+
+.comment-detail__author-more-actions {
+	border-top: 1px solid lighten( $gray, 30% );
+	display: flex;
+	flex-flow: row wrap;
+	padding: 16px 8px;
+}
+
+.comment-detail__author-more-element {
+	box-sizing: border-box;
+	display: block;
+	flex-shrink: 0;
+	flex-grow: 0;
+	overflow: hidden;
+	padding: 8px;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+	width: 100%;
+
+	@include breakpoint( ">660px" ) {
+		width: 50%;
+	}
+}
+
+a.comment-detail__author-more-element {
+	color: $gray;
+	cursor: pointer;
+	fill: $gray;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail__author-more-element-author {
+	display: flex;
+	flex-flow: row nowrap;
+
+	.comment-detail__author-avatar {
+		height: 24px;
+		width: 24px;
+	}
+
+	.comment-detail__author-info {
+		line-height: 1.2;
+		padding-left: 8px;
+	}
+}
+
+.comment-detail.author-is-blocked .comment-detail__author-more-element-block-user .gridicon {
```

Comment:
I think we can simplify this a bit, as the specificity here is quite high. For example if we gave the gridicon a classname, and moved the modifer closer:
```jsx
.comment-detail__block-icon.is-blocked {
    color: $alert-red;
    fill: $alert-red;
}
```